src/Provider/GoogleMaps/Tests/GoogleMapsTest.php

```diff
@@ -501,6 +523,8 @@ public function testGeocodeWithColloquialAreaComponent()
 
     public function testGeocodeWithWardComponent()
     {
+        $this->markTestSkipped('This address no longer has a ward.');
```

Comment:
Not sure what I should do with this test. I've marked it as skipped. 

I tried to find an address with a ward, tried a bunch but could not find one.

Reply:
Indeed it would be nice to find another address with a ward property!

Any idea why the test fails now ?
Does this location not have a ward property anymore ?
Is ward still a property ?

The address returned for this location seems to be `1 Kinkakujichō, Kita-ku, Kyōto-shi, Kyōto-fu 603-8361, Japon`.
"Kita-ku" seems to still be part of the address ; what property is it now ?

Reply:
Yep, it no longer has a ward. I'll have a look at the docs and see if I can see anything regarding wards.

Reply:
Result now: 

```php
Geocoder\Provider\GoogleMaps\Model\GoogleAddress::__set_state(array(
   'id' => 'ChIJnT1kwyCoAWAR-d2HQrYxlTs',
   'locationType' => 'ROOFTOP',
   'resultType' => 
  array (
    0 => 'premise',
  ),
   'formattedAddress' => '1 Kinkakujichō, Kita-ku, Kyōto-shi, Kyōto-fu 603-8361, Japan',
   'streetAddress' => NULL,
   'intersection' => NULL,
   'political' => 'Japan',
   'colloquialArea' => NULL,
   'ward' => NULL,
   'neighborhood' => NULL,
   'premise' => '1',
   'subpremise' => NULL,
   'naturalFeature' => NULL,
   'airport' => NULL,
   'park' => NULL,
   'pointOfInterest' => NULL,
   'establishment' => NULL,
   'subLocalityLevels' => 
  Geocoder\Model\AdminLevelCollection::__set_state(array(
     'adminLevels' => 
    array (
      1 => 
      Geocoder\Model\AdminLevel::__set_state(array(
         'level' => 1,
         'name' => 'Kita-ku',
         'code' => 'Kita-ku',
      )),
      2 => 
      Geocoder\Model\AdminLevel::__set_state(array(
         'level' => 2,
         'name' => 'Kinkakujichō',
         'code' => 'Kinkakujichō',
      )),
    ),
  )),
   'partialMatch' => false,
   'coordinates' => 
  Geocoder\Model\Coordinates::__set_state(array(
     'latitude' => 35.0393986,
     'longitude' => 135.7293744,
  )),
   'bounds' => 
  Geocoder\Model\Bounds::__set_state(array(
     'south' => 35.0391937,
     'west' => 135.7289647,
     'north' => 35.0397866,
     'east' => 135.7295236,
  )),
   'streetNumber' => NULL,
   'streetName' => NULL,
   'subLocality' => 'Kita-ku',
   'locality' => 'Kyōto-shi',
   'postalCode' => '603-8361',
   'adminLevels' => 
  Geocoder\Model\AdminLevelCollection::__set_state(array(
     'adminLevels' => 
    array (
      1 => 
      Geocoder\Model\AdminLevel::__set_state(array(
         'level' => 1,
         'name' => 'Kyōto-fu',
         'code' => 'Kyōto-fu',
      )),
    ),
  )),
   'country' => 
  Geocoder\Model\Country::__set_state(array(
     'name' => 'Japan',
     'code' => 'JP',
  )),
   'timezone' => NULL,
   'providedBy' => 'google_maps',
))
```

Reply:
`Kita-ku` seems to be admin level 1, and the sublocality.

Reply:
There's no reference to wards in the docs (at least, anymore).
https://developers.google.com/maps/documentation/geocoding/intro#Types

Reply:
I've tried a bunch more addresses but have no luck finding a ward.

Reply:
Just remove the `testGeocodeWithWardComponent` test then :)

Reply:
Done :)